content/docs/user-guide/managing-external-data.md

```diff
@@ -4,22 +4,21 @@ There are cases when data is so large, or its processing is organized in a way
 that you would like to avoid moving it out of its external/remote location. For
 example from a network attached storage (NAS) drive, processing data on HDFS,
 running [Dask](https://dask.org/) via SSH, or having a script that streams data
-from S3 to process it. External outputs and
+from S3 to process it. External <abbr>outputs</abbr> and
 [external dependencies](/doc/user-guide/external-dependencies) provide a way for
-DVC to control data outside of the <abbr>project</abbr> directory.
+DVC to track data outside of the <abbr>project</abbr>.
 
 ## Description
 
-DVC can track files on an external storage with `dvc add` or specify external
-files as <abbr>outputs</abbr> for
-[DVC-files](/doc/user-guide/dvc-files-and-directories) created by `dvc run`
-(stage files). External outputs are considered part of the DVC project. DVC will
-track changes in them and reflect this in the output of `dvc status`.
+DVC can track files on an external location with `dvc add` or specify external
+files or directories as outputs for `dvc.yaml` files. External outputs are
+considered part of the (extended) DVC project: DVC will track changes in them,
+and reflect this in `dvc status` for example.
 
 Currently, the following types (protocols) of external outputs (and
 <abbr>cache</abbr>) are supported:
 
-- Local files and directories outside of your <abbr>workspace</abbr>
+- Local files and directories outside the <abbr>workspace</abbr>
 - SSH
 - Amazon S3
 - Google Cloud Storage
```

Comment:
Is this list still complete? The External Deps page also has Azure. Any other one missing in either doc?

Reply:
Cc @efiop perhaps you remember off the top of your head? Thanks

Reply:
HTTP and WebDav(s)

Reply:
Thanks

Reply:
@jorgeorpinel Sorry for confusion, WebDav is not supported as a dependency. HTTP(s) is though.

Reply:
All good, I haven't updated this 😋

Reply:
Wait. You can setup an HTTP external cache? I thought HTTP remotes were always read-only.

I also don't see a `cache.http` field in https://dvc.org/doc/command-reference/config#cache. I see `cache.azure` though, is that the one missing here?

Reply:
OK I'm doing a bunch of changes again so I'll resolve this and just ask you to review the whole page again. Thanks!